dropwizard-parent-pom/pom.xml

```diff
@@ -34,11 +34,14 @@
     <dropwizard.api.key.bundle.version>1.0.0</dropwizard.api.key.bundle.version>
     <dropwizard.configurable.assets.bundle.version>1.0.5</dropwizard.configurable.assets.bundle.version>
     <dropwizard.json.log.bundle.version>1.0.2-0</dropwizard.json.log.bundle.version>
-    <dropwizard.metrics.datadog.ecs.version>0.3.0</dropwizard.metrics.datadog.ecs.version>
+    <dropwizard.metrics.datadog.ecs.version>0.4.0</dropwizard.metrics.datadog.ecs.version>
     <dropwizard.redirect.bundle.version>1.0.5</dropwizard.redirect.bundle.version>
     <dropwizard.template.config.version>1.4.0</dropwizard.template.config.version>
     <dropwizard.version.bundle.version>1.0.0</dropwizard.version.bundle.version>
     <dropwizard.webjars.bundle.version>1.0.0</dropwizard.webjars.bundle.version>
+
+    <environment.bundle.version>0.3.0</environment.bundle.version>
```

Comment:
I think we typically call this `mainstreethub.environment.version` and `mainstreethub.time.version` in other pom files.